Marshal/UnmarshalUpdating.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  M A R S H A L
+//
+//       ()
+//       /\
+//  ()--'  '--()
+//    `.    .'
+//     / .. \
+//    ()'  '()
+//
+//
+
+
+import Foundation
+
+
+public protocol UnmarshalUpdating {
```

Comment:
Small nit: vertical whitespace is inconsistent with Marshaling.swift (above and below the `import`)